serverless.yml

```diff
@@ -1,5 +1,12 @@
 service: cla-bot
 
+resources:
```

Comment:
thanks @maoo - well spotted, I really should have tested a green-field deployment!